operator/deploy/crds/osb_v1alpha1_ansibleservicebroker_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: osb.openshift.io/v1alpha1
```

Comment:
It's worth noting that I brought the apiVersion + kind to look more like what we had done for template service broker.